### ISSUE TITLE
thumbnail images should have alt=""

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#724]
As a screen-reader user, it's repetitive and distracting to have alt tags for images that simply repeat the same content that's already in the text. 

We should set `alt=""` for the thumbnail images in courses lists like https://ocw.mit.edu/collections/environment/ -- similar to what we do on the course cards. 

_Originally posted by @pdpinch in https://github.com/mitodl/ocw-hugo-themes/issues/721#issuecomment-1145905027_
